tests/e2e/support/objects/app-files/resource/index.ts

```diff
@@ -245,4 +252,24 @@ export class Resource {
     await checkThatFileVersionIsNotAvailable({ ...args, page: this.#page })
     await this.#page.goto(startUrl)
   }
+
+  async changePagePersonalSpace(args: Omit<changePageArgs, 'page'>): Promise<void> {
+    await changePagePersonalSpace({ ...args, page: this.#page })
+  }
+
+  async changeItemsPerPage(args: Omit<changeItemsPerPageArgs, 'page'>): Promise<void> {
+    await changeItemsPerPage({ ...args, page: this.#page })
+  }
+
+  async getFileListFooterText(): Promise<string> {
```

Comment:
```suggestion
  getFileListFooterText(): Promise<string> {
```
like this or we need async await for it?

Reply:
no we don't, Nice catch thanks